### PR TITLE
Update configuration-management-reference-microsoft-defender-antiviru…

### PR DESCRIPTION
**Pull request by msbemba**
…s.md

Added Microsoft defender antivirus to the applied to section.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5471

File: microsoft-365/security/defender-endpoint/configuration-management-reference-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ ms.topic: article
 **Applies to:**
 
 - [Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/)
+-  Microsoft Defender Antivirus
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configuration-management-reference-microsoft-defender-antivirus.md]:
```suggestion
- Microsoft Defender Antivirus
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**